core/src/main/scala/org/apache/spark/storage/BlockManagerMaster.scala

```diff
@@ -14,7 +14,6 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
```

Comment:
Shall we revert this irrelevant change to focus on the new contribution?

Reply:
Sure